maas/resource_maas_network_interface_bridge_test.go

```diff
@@ -0,0 +1,45 @@
+package maas_test
+
+import (
+	"terraform-provider-maas/maas/testutils"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const TestAccResourceMaasNetworkInterfaceBridgeConfig_basic = `
```

Comment:
Please perform the same refactoring as has been explained to bond acceptance tests